src/Pages/Cart/CartList.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+import CartItem from './CartItem';
+
+const CartList = ({ cartItem, handleCount, handlePlus, handleMinus, handleDelete, deleteAll }) => {
+  const history = useHistory();
+  return (
+    <Container>
+      <h3 className="cartTitle">장바구니</h3>
+      <CartContainer>
+        {cartItem.length !== 0 ? ( //true
```

Comment:
불리안 값을 이용해 아래와 같이 작성하실 수 있겠네요! 
```suggestion
        {cartItem.length ? 
```

Reply:
{cartItem.length ? 말고 {cartItem ? 이렇게 해도 되나요?

Reply:
수정 완료했습니다!